package/MDAnalysis/analysis/contacts.py

```diff
@@ -536,7 +536,6 @@ def q1q2(u, selection='all', radius=4.5,
 # until then it remains because we don't have the functionality
 # elsewhere.
 
-@deprecate(new_name="Contacts", message="This class will be removed in 0.17")
```

Comment:
having the deprecate at the top of the class made sphinx think this is a function and not a class. Moving it to the `__init__` function fixed that. Bonus we can also use the recent sphinx version now.